hatice-ecevit-2016400138/assignment3/Dict.java

```diff
@@ -0,0 +1,84 @@
+import java.util.HashMap;
+
+public class Dict {
+    public HashMap<Character,DictNode> letters = new HashMap<Character, DictNode>();

```

Comment:
It is good practice to make your fields private - it stops the data structure being interfered with by other classes. See https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)